lib/opencensus/trace/config.rb

```diff
@@ -12,9 +12,71 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+require "opencensus/trace/samplers"
+
 module OpenCensus
   module Trace
-    class Config
+    ##
+    # The OpenCensus Trace configuration. See Trace#configure for more info.
+    #
+    Config = Common::Config.new do |config|
+      config.add_option! :default_sampler, Samplers::DEFAULT do |value|
+        value.respond_to? :call
+      end
+      config.add_option! :default_max_attributes, 32
+      config.add_option! :default_max_stack_frames, 32
+      config.add_option! :default_max_annotations, 32
+      config.add_option! :default_max_message_events, 128
+      config.add_option! :default_max_links, 128
+      config.add_option! :default_max_string_length, 1024
+    end
+
+    class << self
+      ##
+      # Configure OpenCensus Trace. These configuration fields included
+      # parameters governing sampling, span creation, and exporting.
+      #
+      # Generally, you should configure this once at process initialization,
+      # but it can be modified at any time.
+      #
+      # Example:
+      #
+      #     OpenCensus::Trace::configure do |config|
```

Comment:
Do we prefer this method of class method calling vs. `OpenCensus::Trace.configure`

Reply:
No, that was a typo. Fixed.